pkgs/development/libraries/libfive/default.nix

```diff
@@ -96,7 +96,9 @@ stdenv.mkDerivation {
     "libfive.stdlib"
   ];
 
-  passthru.updateScript = unstableGitUpdater { };
+  passthru.updateScript = unstableGitUpdater {
+    tagFormat = "";
+  };
```

Comment:
can you explain why you made this change? Is the default something else than `""`?

Reply:
@pbsds the default is `”*”`, which for some reason causes the script to pick up a tag from 2017 and attempt to parse it, which fails because the tag format is not numeric. I don’t know why this wasn’t encountered previously, I didn’t bother looking to see if there was a change in the updater script.